public/static-checkout/index.html

```diff
@@ -0,0 +1,148 @@
+                  <div class="product-line-item">
+                    <h4>Title HERE</h4>
+                    <p>Verified Certificate</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </section>
+          <!-- Summary -->
+          <div class="checkout-section" role="region" aria-labelledby="summary-heading" aria-live="polite">
+            <h5>Summary</h5>
+            <p class="col-10">Price</p>
+            <span>$350.00</span>
+            <!-- Coupon form  -->
+            <p>Add coupon code (optional)</p>
+            <form style="display: inline-flex; margin-bottom: 1rem;">
+              <input style="margin-right: 0.5rem; width: 12rem;" type="text" name="couponcode" aria-describedby="">
+              <button class="submit-button" style="font-size: 1rem; width: 6.5rem; font-weight: 400;">Apply</button>
+            </form>
+            <h4 class="col-10">TOTAL</h4>
+            <span><strong>$350.00</strong></span>
+          </div>
+          <!-- Order Details -->
+          <div class="checkout-section">
+            <h5>Order Details</h5>
+            <p>After you complete your order you will be automatically enrolled in the verified track of the course.</p>
+          </div>
+        </div>
+
+        <!-- Checkout  -->
+        <div class="checkout col-7">
+          <section aria-label="Payment Details">
+            <!-- Payment Method  -->
+            <div class="checkout-section">
+              <h5>Select Payment Method</h5>
+              <div class="payment-method">
+                <button class="active-button"><img src="assets/accepted-card-logos.png" alt="card-logos"></button>
+                <button><img src="assets/paypal-logo.png" alt="paypal-logo"></button>
+              </div>
+            </div>
+            <!-- Checout Form  -->
+            <div class="card">
+              <div class="checkout-section">
+                <h5>Card Holder Infomation</h5>
+                  <form>
+                    <div class="col-left">
+                      First name (required)<br>
+                      <input type="text" name="firstname"><br>
+                    </div>
+                    <div class="col-right">
+                      Last name (required)<br>
+                      <input type="text" name="lastname"><br>
+                    </div>
+
+                    <div class="col-left">
+                      Address (required)<br>
+                      <input type="text" name="address"><br>
+                    </div>
+                    <div class="col-right">
+                      Suite/Apartment Number<br>
+                      <input type="text" name="address2"><br>
+                    </div>
+
+                    <div class="col-left">
+                      City (required)<br>
+                      <input type="text" name="city"><br>
+                    </div>
+                    <div class="col-right">
```

Comment:
country/expiration month and expiration year are dropdowns on the mfe.
i think making at least the expiration fields dropdown here would be a good idea if a wrong format month/year would cause the payment to fail.  
Not sure about country, that's more important for the sdn check than the payment itself, so probably a user inputted country is fine (though we'll get some different spellings)

Reply:
Good call, I totally forgot to have them as dropdowns - updated on the next commit